src/transformers/image_transforms.py

```diff
@@ -115,10 +117,11 @@ def rescale(
     return rescaled_image
 
 
+@vision_required
```

Comment:
How is this different than having `requires_backend(["vision"])` as the first line of the function?

Reply:
To be honest, I just saw the `tf_required` and copied from there. I'll remove the decorator and add those instead 👍

Reply:
Opened a draft PR here: https://github.com/huggingface/transformers/pull/20715